tests/gen/gen_auto_configurator.go

```diff
@@ -89,10 +89,10 @@ func (c *typeUsageCounters) getNextTypeIndForField(secondaryType string, field *
 type relationUsage struct {
 	// counter is the number of primary documents that have been used for the relation.
 	counter int
-	// minAmount is the minimum number of primary documents that should be used for the relation.
-	minAmount int
-	// maxAmount is the maximum number of primary documents that should be used for the relation.
-	maxAmount int
+	// minSecDocPerPrim is the minimum number of primary documents that should be used for the relation.
+	minSecDocPerPrim int
```

Comment:
suggestion: I'm not sure the loss of characters is worth it here, and `minSecondaryDocsPerPrimary` would be worth the extra length?  Or maybe add an `s` to `Doc`: `minSecDocsPerPrim`?